tests/integration/linux_bridge_test.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import yaml
+
+from libnmstate import netapplier
+
+from .testlib import statelib
+from .testlib.statelib import INTERFACES
+
+
+IPV4_ADDRESS1 = '192.0.2.251'
+IPV4_ADDRESS2 = '192.0.2.252'
+
+LINUX_BRIDGE_YAML_BASE = """
+interfaces:
+  - name: linux-br0
+    type: linux-bridge
+    state: up
+    bridge:
+      options:
+        group-forward-mask: 0
+        mac-ageing-time: 300
+        multicast-snooping: true
+        stp:
+          enabled: true
+          forward-delay: 15
+          hello-time: 2
+          max-age: 20
+          priority: 32768
+      port:
+        - name: eth1
```

Comment:
It would be geat to also test more than one port, adding/removing ports to an existing bridge, setting up a VLAN on top of a bridge etc. This does not need to happen in this PR but it makes me wonder if we should start writing a high-level test plan (maybe with user stories) and then see if we cover all the scenarios.

Reply:
I agree. I think we can add such a plan in the jira issues in advance and then implement them. New features should have these in advance, as part of the feature story.
Anyway, this discussion is part of our retrospection/planning.